share/cmake/modules/Findyaml-cpp.cmake

```diff
@@ -29,8 +29,24 @@ if(NOT OCIO_INSTALL_EXT_PACKAGES STREQUAL ALL)
     set(_yaml-cpp_REQUIRED_VARS yaml-cpp_LIBRARY)
```

Comment:
This change seems like it slipped in from Patrick's CMake work which we resolved. Should likely revert this file.

Reply:
Will do.

Reply:
There's still changes being made to this file in the most recent commit. I think the state the file is in master currently is working well so would be good to leave it unchanged.

Reply:
I had tried to revert my original local commit but that might have interfere with the merge from master. Next commit will replace these files with their versions in master.